tartiflette_aiohttp/__init__.py

```diff
@@ -116,6 +120,7 @@ def register_graphql_handlers(
         subscription_ws_endpoint {Optional[str]} -- Path part of the URL the WebSocket GraphQL subscription endpoint will listen on (default: {None})
         graphiql_enabled {bool} -- Determines whether or not we should handle a GraphiQL endpoint (default: {False})
         graphiql_options {dict} -- Customization options for the GraphiQL instance (default: {None})
+        engine_modules: {Optional[List[Union[str, Dict[str, Union[str, Dict[str, str]]]]]]} -- Module to import (default:{None})
```

Comment:
Put that line close to the other `engine_` parameters. (e.g. `engine_sdl `, `engine_schema_name`)